bcdaworker/main.go

```diff
@@ -127,7 +132,13 @@ func processJob(j *que.Job) error {
 			return err
 		}
 	} else {
-		err = encryptData(stagingPath, payloadPath, fileName, exportJob)
+		encryptionEnabled := utils.GetEnvBool("ENABLE_ENCRYPTION", true)
```

Comment:
To ensure we haven't missed anything, can you link to the ticket for final removal of feature flags once we have iterated on the code with feature flags included?
Obviously, this implementation will occur in the future but want to make sure we don't forget about it.

Reply:
yep will create future ticket for full encryption removal.